hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -236,6 +237,8 @@ public class HoodieTableConfig extends HoodieConfig {
 
   private static final String TABLE_CHECKSUM_FORMAT = "%s.%s"; // <database_name>.<table_name>
 
+  private static Set<String> metadataPartition = null;
+
```

Comment:
keep the result in static variable is not a good way, either we keep the code as it is either we introduce some cache tools like the guava interns and initialize it when building the table config(in the constructor).

Reply:
Thanks for review, static is not a good way, Can we keep the result in a Set, and init when the first get is called?